### PR TITLE
Upgrade Quarkus to version 2.1.2

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Review by radtriste — APPROVED** [review]

File: .github/workflows/apps-pr.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     concurrency:
       group: apps_pr-${{ github.head_ref }}
       cancel-in-progress: true
-    timeout-minutes: 120
+    timeout-minutes: 180
```
Comment (ricardozanini) [inline .github/workflows/apps-pr.yml:22]: :sweat_smile:

**Comment by ricardozanini** [comment]
Have you run the script to update the quarkus dependencies?

**Comment by cristianonicolai** [comment]
> Have you run the script to update the quarkus dependencies?

nope

**Comment by ricardozanini** [comment]
The only change I see is this:

```
-    <version.io.smallrye.reactive.mutiny-vertx-web-client>2.12.0</version.io.smallrye.reactive.mutiny-vertx-web-client>
+    <version.io.smallrye.reactive.mutiny-vertx-web-client>2.11.0</version.io.smallrye.reactive.mutiny-vertx-web-client>
`` 

This is weird because indeed they are using `2.11.0`: https://github.com/quarkusio/quarkus/blob/2.1.2.Final/bom/application/pom.xml#L56

Maybe I slipped this version naming when I did the dependencies BOM?

**Comment by cristianonicolai** [comment]
@ricardozanini np, I reverted that

**Pull request merged**